jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/StandardDescriptorProcessorTest.java

```diff
@@ -137,7 +138,18 @@ public void testDuplicateServletMappingsFromDescriptors(WorkDir workDir) throws
         wac.setBaseResourceAsPath(docroot);
         wac.setDescriptor(webXml.toURI().toURL().toString());
         wac.start();
-        assertEquals("other", wac.getServletHandler().getServletMapping("/").getServletName());
+        ServletMapping[] mappings = wac.getServletHandler().getServletMappings();
+        ServletMapping mapping = null;
+        for (ServletMapping m : mappings)
+        {
+            if (m.containsPathSpec("/"))
+            {
+                assertEquals(null, mapping);
+                mapping = m;
```

Comment:
Add `assertNull(mapping, "Found duplicate mapping")` before the `mapping = m;`

if `mapping` was already set, this is a test failure (you just found duplicate mappings).

Reply:
Again, this is exactly what line 147 is doing.